layouts/partials/footer.hbs

```diff
@@ -40,6 +58,49 @@
     e.preventDefault();
     window.scrollTo(0, 0);
   });
+
+  (async function () {
```

Comment:
Also this should be move in main.js and be ES5

Reply:
Btw does it mean I should use XHR and not fetch at all ?

Reply:
Probably yes. The frontend JS is ES5

Reply:
Ok, i'll get rid of that

Reply:
Please rebase from upstream/master and move any JS to static/javascript/main.js